src/main/features/response/views/macro/expandableSection.njk

```diff
@@ -0,0 +1,13 @@
+{% macro expandableSection(label = '', content = '', limit = 400, id='') %}
```

Comment:
Empty default values for label, content and what is more ID does not look like good idea. When these are not provided we should fail fast.

Have you tried to generate the ID by some sort of function? That would take the requirement from clients to pass it as this is very internal thing.

Reply:
I will remove the default values so that it fails fast. Adding a function to generate ids seems a bit over engineering